assets/airports/airportLoadList.json

```diff
@@ -283,6 +283,12 @@
         "name": "Miami International Airport",
         "premium": false
     },
+    {
+        "icao": "kmht",
+        "level": "easy",
```

Comment:
With the current traffic levels, it falls under 'medium'. Between 20 and 40 acph is 'easy'

Reply:
✔️